user-interface/src/main/java/life/qbic/datamanager/Application.java

```diff
@@ -37,6 +40,7 @@
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jms.annotation.JmsListener;
 
 /**
  * The entry point of the Spring Boot application.
```

Comment:
NIT: A lot of imports in the Application class are now unnecessary after your cleanup.

Reply:
will clean up!